Update report_metrics.md (#2134)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/report_metrics.md

    calculation: (Button 2 Clicks) / (Impressions)
    tags:
      - In-App Message

>   - name: Body Click

Reviewer: An IAM section for body clicks already seems to exist above

Author: good catch thanks!

>     description: Total number of clicks on a Push Story Notification not on an action button, or a non-HTML In App Message close button.

Author: ```suggestion
    description: Total number of clicks on a Push Story Notification, excluding action buttons, or a non-HTML In App Message close button.
```

>     description: Total number of clicks on a Push Story Notification, excluding action buttons, or a non-HTML In App Message close button.

Author: ```suggestion
    description: Push Story Notifications record a body click when the message is clicked. It will not included a message being expanded, or action button clicks. Non-HTML In App Messages will record a Body Click when the close button is clicked.
```

>     description: Push Story Notifications record a body click when the message is clicked. It will not included a message being expanded, or action button clicks. Non-HTML In App Messages will record a Body Click when the close button is clicked.

Author: ```suggestion
    description: Push Story Notifications record a body click when the message is clicked. It will not be recorded when a message is expanded, or action button clicks. Non-HTML In App Messages will record a Body Click when the close button is clicked.
```
